src/abstractdataframe/abstractdataframe.jl

```diff
@@ -422,17 +425,20 @@ describe(df)
 """
 StatsBase.describe(df::AbstractDataFrame; kwargs...) = describe(stdout, df; kwargs...)
 function StatsBase.describe(io::IO, df::AbstractDataFrame; stats::AbstractVector{Symbol} = 
```

Comment:
I had missed this in the previous PR, but the `io` argument isn't used now. It should be removed (and the two methods merged), with a deprecation like this in deprecated.jl:
```julia
 @deprecate StatsBase.describe(io::IO, df::AbstractDataFrame; stats::AbstractVector{Symbol}=[:mean, :min, :median, :max, :nunique, :nmissing, :eltype]) show(io, describe(df, stats=stats))
```

Reply:
So we should have just one function definition, `StatsBase.describe(df::AbstractDataFrame, stats...)`?

Reply:
Yes.

Reply:
Fixed this on the second commit below. Should be ready, then.